src/mudlet.cpp

```diff
@@ -2067,9 +2067,12 @@ void mudlet::slot_show_help_dialog_download()
 
 void mudlet::slot_show_about_dialog()
 {
-    auto pDlg = new dlgAboutDialog( this );
-    pDlg->raise();
-    pDlg->show();
+    if( ! mpAboutDlg ) {
+        mpAboutDlg = new dlgAboutDialog( this );
```

Comment:
I had a look through [qt creator](https://github.com/qtproject/qt-creator/search?utf8=%E2%9C%93&q=Qt%3A%3AWA_DeleteOnClose&type=) and they use it plenty, so nevermind me :)